docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -1409,19 +1410,21 @@ UDP
 
   * TcpUpstreamConnected - When TCP Proxy filter has successfully established an upstream connection.
   * TcpPeriodic - On any TCP Proxy filter periodic log record.
-  * TcpConnectionEnd - When a TCP connection is ended on TCP Proxy filter.
+  * TcpConnectionStart - When a TCP connection is started by a listener.
+  * TcpConnectionEnd - When a TCP connection is ended on TCP Proxy filter or a listener.
```

Comment:
I just realized `TcpConnectionEnd` is reused for TcpProxy and listener. IMHO, it is fine from TcpProxy/Listener perspective but it is more intuitive to specify Downstream/Upstream and also consistent with the existing http stream style?